example/server.js

```diff
@@ -26,23 +26,20 @@ var validate = function (decoded, request, callback) {
 
   // do your checks to see if the person is valid
   if (!people[decoded.id]) {
-    return callback(null, false);
+    return { valid: false };
```

Comment:
shouldn't this be `isValid` instead of `valid`?

Reply:
fixed and rebased